### ISSUE TITLE
Re-do the UI of suggested passages.

### ISSUE THREAD

**Issue by johnraftery** [johnraftery/BibleTrainer#44]
For example, it's terrible that pressing back doesn't cancel the drop-down menu of topics. But I think we should have a proper chat about how this whole thing should work.

**Comment by jamesdoc** [comment]
Yes. I have a napkin sketch of an improved flow for this already.

**Comment by jamesdoc** [comment]
Here is an updated concept flow for the Add Passage / Suggested Passage UI:

<img width="1340" alt="screenshot 2019-02-17 at 13 12 05" src="https://user-images.githubusercontent.com/790343/52913397-b47bea80-32b5-11e9-8133-8b07976977eb.PNG">

The idea is to merge them into one screen; regardless of if you are looking up a passage or picking from a library you are 'adding a passage'.

The confirm step needs a little bit more work to show the _whole_ passage that has been pulled in. Will give that some more thought over the coming days.

Once the flow has been completed the user should be returned to the passage list _not_ the add passage screen.

**Comment by johnraftery** [comment]
I really like it! One comment though: if I want to add several passages in one go (e.g. sermon series at church has just been published), to be returned to the passage list after adding and then have to start the flow again from the start would be annoying.

**Comment by jamesdoc** [comment]
After a verse has been added BibleTrainer should 'remember' the last saved passage, and then offer that book and chapter automagically next time the user wants to add a passage
